.circleci/config.yml

```
@@ -175,13 +83,12 @@ jobs:
             - node_modules
 
   test-unit:
-    <<: *defaults_working_directory
-    <<: *defaults_docker_node
+    executor: default-docker
     steps:
+      - checkout
       - run:
           name: Install general dependencies
           command: *defaults_Dependencies
-      - checkout
       - restore_cache:
           keys:
             - dependency-cache-{{ checksum "package.json" }}
```

```suggestion
            - dependency-cache-{{ checksum "package-lock.json" }}
```
For the same reason as above. (Note: I know you didn't change this, but it seemed like a good a time as any to update this in line with the other usage.)